addons/web/static/src/legacy/js/widgets/model_field_selector.js

```diff
@@ -85,69 +60,66 @@ var ModelFieldSelector = Widget.extend({
         this.options = _.extend({
             order: 'string',
             readonly: true,
-            filters: {},
             fields: null,
-            filter: function () {return true;},
-            followRelations: true,
-            debugMode: false,
-            showSearchInput: true,
+            debugMode: false
         }, options || {});
-        this.options.filters = _.extend({
-            searchable: true,
-        }, this.options.filters);
 
-        if (typeof this.options.followRelations !== 'function') {
-            this.options.followRelations = this.options.followRelations ?
-                function () {return true;} :
-                function () {return false;};
-        }
-
-        this.pages = [];
         this.dirty = false;
+        this.popOver = new ModelFieldSelectorPopOver(this, model, chain, options);
```

Comment:
I think the `o` in popover should never be capitalized.

Reply:
I can change that